### PR TITLE
Adding optional support for special and character and optional oxford…

### PR DESCRIPTION
**Pull request by rockcitydiesel**
… comma

### CODE REVIEW

File: test/enumerate.spec.js [hunk 1]
```diff
@@ -33,4 +33,14 @@ describe('enumerate', function() {
     runTests(enumerate, tests)
   })
 
+  it('should take in optional parameters', function() {
+      const test = [
```
Comment (albertywu) [inline test/enumerate.spec.js]: `tests` (see `L44`)

**Comment by albertywu** [comment]
tests are failing. Try running:

`npm install`

and then 

`npm test`

**Comment by rockcitydiesel** [comment]
yeah. i couldn't get it to run. your code has some weird syntax haha. i'll
try again.

**Comment by rockcitydiesel** [comment]
[enumerate (master)]$ npm test

> enumerate-words@2.0.1 test /Users/vtran/Projects/enumerate
> ./node_modules/mocha/bin/mocha

/Users/vtran/Projects/enumerate/test/enumerate.spec.js:5
const runTests = (fn, tests) => tests.forEach(([args, output]) =>
expect(fn.apply(this, args)).toBe(output))
                                               ^

SyntaxError: Unexpected token [
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at
/Users/vtran/Projects/enumerate/node_modules/mocha/lib/mocha.js:220:27
    at Array.forEach (native)
    at Mocha.loadFiles
(/Users/vtran/Projects/enumerate/node_modules/mocha/lib/mocha.js:217:14)
    at Mocha.run
(/Users/vtran/Projects/enumerate/node_modules/mocha/lib/mocha.js:485:10)
    at Object.<anonymous>
(/Users/vtran/Projects/enumerate/node_modules/mocha/bin/_mocha:403:18)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:429:10)
    at startup (node.js:139:18)
    at node.js:999:3
npm ERR! Test failed.  See above for more details.

**Comment by albertywu** [comment]
Try rebasing off of master? I updated the test suite to be more standard :)

**Comment by bcherny** [comment]
@rockcitydiesel Be sure to use Node 6.0+

**Comment by rockcitydiesel** [comment]
[enumerate (vinhbranch)]$ node --version
v5.4.1

**Pull request closed without merging**